dist/MoreMenu.js

```diff
@@ -576,6 +576,33 @@ $(function () {
     });
     return html;
   }
+  /**
+   * The #p-cactions element is always in the DOM, so we can use it
+   * to create our own menu without worrying about HTML or class changes.
+   *
+   * @throws {Error} if the element has been unexpectedly removed from the DOM
+   *  so that appropriate action can be taken.
+   * @return {string}
+   */
+
+
+  function makeMenuHtml() {
+    var $menu = $('#p-cactions').clone(true);
+
+    if ($menu.length) {
+      $menu.attr('id', "p-".concat(parentKey));
+      $menu.addClass(["mw-portlet-".concat(parentKey), "mm-".concat(parentKey), 'mm-tab']).removeClass('mw-portlet-cactions').attr('aria-labelledby', "p-".concat(parentKey, "-label"));
+      $menu.find('input[type="checkbox"]').attr('id', "p-".concat(parentKey, "-checkbox"));
+      $menu.find('label').attr('id', "p-".concat(parentKey, "-label"));
+      $menu.find('label').text(msg(parentKey));
+      $menu.find('ul').addClass('mm-menu').html(html);
+      return $menu[0].outerHTML;
+    } else {
+      // Throwing an error will notify Wikimedia developers that they
+      // broke this script by showing up in our error logs.
+      throw new Error('MoreMenu: p-cactions is not in DOM. Expection with Vector maintaining team not met.');
```

Comment:
Clever! So this will show in logstash?

Also, typo: `expection`

Reply:
Yep!

This is a good idea for any popular and maintained script as it will show up in https://grafana.wikimedia.org/d/000000566/overview?orgId=1&viewPanel=16 which will block the train if we go into red zone. It should also show up on the beta cluster if you have a script to update it there, which gives a small chance that any issue will be picked up before the train begins to roll out.

It's also a nice way to setup an informal but enforceable contract with us :)